src/Stratis.Bitcoin/P2P/PeerConnectorAddNode.cs

```diff
@@ -52,20 +52,20 @@ public PeerConnectorAddNode(NodeSettings nodeSettings, IPeerAddressManager peerA
         /// <inheritdoc/>
         public override NetworkAddress FindPeerToConnectTo()
         {
-            PeerAddress peer = null;
-
             foreach (var endPoint in this.NodeSettings.ConnectionManager.AddNode)
             {
-                peer = this.peerAddressManager.FindPeer(endPoint.MapToIpv6());
+                PeerAddress peer = this.peerAddressManager.FindPeer(endPoint.MapToIpv6());
```

Comment:
You can create a variable to hold the value of `endPoint.MapToIpv6()`.

Reply:
OK yes, I have moved MapToIpv6() to the Match method so that it only happens in place :)